diff --git a/Android.bp b/Android.bp
index 3e82eff..550133f 100644
--- a/Android.bp
+++ b/Android.bp
@@ -35,8 +35,10 @@ rust_defaults {
 rust_test_host {
     name: "shared_library_host_test_src_lib",
     defaults: ["shared_library_test_defaults"],
+    // Manually disabled as these tests depend on specific graphics libraries
+    // being available on the machine running the tests.
     test_options: {
-        unit_test: true,
+        unit_test: false,
     },
 }
 
